stm32f0_hal/examples/pwm.rs

```diff
@@ -3,11 +3,22 @@
 extern crate stm32f0_hal as hal;
 
 use hal::pwm;
+use hal::rcc;
 
 fn main() {
-    pwm::init(10000);
-    pwm::set_duty(750);
-    pwm::enable();
-
+    rcc::init();
+    let channel1 = pwm::Pwm::init(pwm::Pin::PC6);
+    let channel2 = pwm::Pwm::init(pwm::Pin::PC7);
+    let channel3 = pwm::Pwm::init(pwm::Pin::PC8);
+    let channel4 = pwm::Pwm::init(pwm::Pin::PC9);
+    channel1.set_frequency(49);
```

Comment:
It's weird that you are only setting the frequency for the channel 1.

Reply:
In fact, you have only one frequency for all the channel of the same TIMER.